calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/views/image_view.py

```diff
@@ -202,6 +211,162 @@ def set_rendering_alpha(self, value: float):
         """Set the alpha channel of the drawings."""
         self.rendering_alpha = value
 
+    def set_draw_indicators(
+        self,
+        board_speed: float,
+        max_allowed_board_speed: float,
+        skew_percentage: float,
+        board_size_percentage: float,
+        rows_linear_error: float,
+        cols_linear_error: float,
+        pct_err_rows: float,
+        pct_err_cols: float,
+        aspect_ratio: float,
+        pan: float,
+        tilt: float,
+        alpha_indicators: float,
+        value: bool,
+    ):
+        """Set values for indicators."""
+        self.current_board_speed = board_speed
+        self.max_board_allowed_speed = max_allowed_board_speed
+        self.skew_percentage = skew_percentage
+        self.board_size_percentage = board_size_percentage
+        self.rows_linear_error = rows_linear_error
+        self.cols_linear_error = cols_linear_error
+        self.pct_err_rows = pct_err_rows
+        self.pct_err_cols = pct_err_cols
+        self.aspect_ratio = aspect_ratio
+        self.pan = pan
+        self.tilt = tilt
+        self.alpha_indicators = alpha_indicators
+        self.is_draw_indicators = value
+
+    def draw_indicators(self, painter: QPainter, display_size):
+        """Draw indicators for speed, skew, size aspect ratio, angles of the detected board."""
+        color_green = QColor(0.0, 255, 0.0, int(255 * self.alpha_indicators))
+        color_red = QColor(255, 0.0, 0.0, int(255 * self.alpha_indicators))
+        # Change color according tot he speed
```

Comment:
tot he

Reply:
thanks, i will fix it

Reply:
Fixed,

Reply:
Whenever you address a comment, please reference the commit here. These commits should be as small as possible, so the review process go smoother

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
This is not the commit that addresses the comment.
In addition, it has not been fixed

Reply:
[ec063e9](https://github.com/tier4/CalibrationTools/pull/220/commits/ec063e916576efd4d43021693ba83ffc22ae1910)